Mods/vcmi/config/vcmi/english.json

```diff
@@ -38,6 +38,7 @@
 	"vcmi.radialWheel.splitUnit" : "Split creature to another slot",
 
 	"vcmi.mainMenu.tutorialNotImplemented" : "Sorry, tutorial is not implemented yet\n",
+	"vcmi.mainMenu.tutorialNotExist" : "Tutorial map is not found\n",
```

Comment:
Please use existing string from H3
`	"core.genrltxt.742" : "The tutorial map file \"tutorial.tut\" is missing from the maps directory.  You cannot start a tutorial without this file.",
`